src/components/Tutorials/NewTutorial/index.js

```diff
@@ -96,106 +111,126 @@ const NewTutorial = ({ viewModal, viewCallback, active }) => {
         })
       : null;
 
+  const list = [];
+  orgList && orgList.map((org) => list.push(org.props.value));
```

Comment:
Learn [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) on JavaScript. It is a cool new JS feature.

Reply:
Thanks for sharing resources, learnt something new.